scripts/linters/html_linter.py

```diff
@@ -108,7 +122,12 @@ def handle_starttag(self, tag, attrs):
         indentation_of_first_attribute = (
             column_number + len(tag) + 2)
         starttag_text = self.get_starttag_text()
-
+        # Ruling out the possibility of None of 'starttag_text' for mypy
```

Comment:
Why can't starttag_text always return a non-None value, and the callers handle the case where there's no tag (and don't call get_starttag_text at all when there's no tag to call it for)?